sound/voice_data.h

```diff
@@ -28,14 +28,34 @@ const uint8_t spSIX[] PROGMEM = {0x0E,0xD8,0xAE,0xDD,0x03,0x0E,0x38,0xA6,0xD2,0x
+const uint8_t spHUNDRED[] PROGMEM = {0x68,0x9a,0x2e,0xcc,0x2d,0xe2,0x20,0x75,0x84,0x0c,0xf7,0x48,0x4c,0x8a,0x61,0xba,0x35,0xd2,0xb1,0x92,0x6b,0x98,0xb0,0xc9,0xc4,0x6e,0xa9,0xf1,0x52,0x27,0xd3,0x04,0xbd,0xd6,0x0d,0xef,0x4c,0xd2,0x8c,0x1c,0x82,0xa3,0x45,0xf1,0x31,0x4b,0x70,0xf5,0x0c,0xdb,0xc3,0x4a,0xc6,0xac,0xca,0x9d,0x38,0x4b,0x9a,0x8e,0x09,0x93,0x6d,0x1c,0xee,0x36,0xd3,0x4d,0x96,0x70,0x3d,0xf7,0x68,0xcf,0x58,0x02,0x38,0x65,0x93,0x01,0xab,0x6c,0x35,0xaf,0x58,0xb2,0xea,0x6a,0xdc,0xbc,0x6a,0xc9,0xba,0x3a,0x72,0xf3,0xb2,0x14,0x9f,0x49,0xdb,0xcd,0x4b,0xd2,0xa2,0x27,0xec,0x38,0xcf,0xa9,0xac,0x89,0x52,0x82,0x1c,0xa9,0x27,0xdb,0x55,0x32,0x02,0x8e,0x2b,0x27,0xe6,0xd4,0xce,0x6e,0x69,0x19,0x99,0x43,0x3b,0xab,0x86,0x65,0x78};
+const uint8_t spTHOUSAND[] PROGMEM = {0x00,0x00,0xca,0xcb,0xdc,0xc3,0xad,0x9c,0x98,0x69,0xfa,0x34,0x73,0x93,0xac,0x80,0x15,0xca,0x82,0x9b,0x9d,0x67,0xa5,0x29,0x69,0x4e,0xce,0xd1,0x39,0xaa,0x68,0x38,0xb9,0x66,0x56,0x9b,0xe2,0xe1,0xe4,0x38,0x91,0xe9,0xb6,0x87,0x9d,0x62,0x7b,0x44,0x38,0x2a,0x4e,0x4c,0xa9,0x3e,0xa6,0x38,0xb8,0xde,0xa7,0x66,0xa5,0x23,0xe5,0x79,0x3e,0x91,0xe1,0xb6,0xd0,0x17,0xd8,0x64,0x78,0xc9,0x22,0xc0,0x96,0x19,0xc2,0x99,0xba,0xcc,0xc3,0x95,0x28,0x57,0xda,0xce,0xca,0xa0,0xec,0x3c,0xe5,0x33,0x3a,0x43,0x8e,0xf3,0xb4,0xe9,0xa8,0x72,0x39,0xca,0x63,0x7e,0xa2,0xc2,0x64,0x19,0x4f,0xea,0xce,0x8a,0xb0,0xc5,0x3c,0x92,0x26,0xd3,0x8c,0x32,0xf1,0xa8,0x99,0x2c,0x33,0x59,0xcc,0xad,0x5c,0x2d,0x43,0x9d,0xb0,0x61,0x2a,0xd7,0x8c,0xb4,0x8c,0xb6,0x61,0xdd,0xac,0xd3,0x09,0x00,0x00,0x3c};
+const uint8_t spMILLION[] PROGMEM = {0x00,0xc8,0x8e,0xcc,0x24,0x32,0x63,0x39,0x87,0xab,0x8a,0xca,0xb0,0x95,0x1c,0xa9,0x2a,0x2a,0xdd,0xf2,0xb2,0x93,0x52,0xef,0x4e,0x3b,0xcb,0x8e,0xca,0x7c,0x26,0xad,0x2c,0x3b,0x98,0xd0,0x9c,0x8a,0xd3,0x1c,0x1f,0x83,0x73,0x22,0x49,0x73,0xbd,0x4f,0xf5,0xac,0x24,0xc5,0xd3,0x21,0x3c,0xbb,0x1c,0x17,0xdf,0xa8,0xec,0xce,0x54,0x5c,0xc2,0xc0,0x75,0xa6,0xcc,0xd6,0x88,0x0b,0xe7,0xea,0x36,0xdb,0x23,0xa9,0x9c,0x7d,0xda,0x6d,0xb7,0xa4,0x08,0x8e,0x6e,0x93,0xdd,0x92,0x2c,0x34,0xab,0x5c,0x75,0x88,0x95,0xce,0xaa,0x52,0x29,0x21,0xd4,0xa2,0xb3,0x22,0x1c,0xbb,0x80,0xd9,0xa9,0x72,0xb3,0x6d,0x7c,0xea,0x37,0xcb,0x55,0x16,0xf3,0x51,0xd8,0x4a,0x51,0xda,0xc8,0x57,0xb4,0xa2,0xdb,0x1d,0x23,0x3f,0x09,0xb5,0x4c,0xb3,0x8d,0xc6,0xc2,0x45,0x3c,0x5c,0x09,0x00,0x00,0x3c};
 const uint8_t spPOINT[] PROGMEM = {0xa0,0xfe,0xa1,0xc3,0xd5,0x25,0x4a,0xc0,0xba,0xad,0x30,0x93,0xc8,0xc9,0x09,0xb6,0x45,0x5c,0xec,0x14,0x27,0x98,0x63,0xc8,0xa6,0xda,0xdc,0x68,0x4e,0x20,0x9a,0xec,0xf0,0x53,0x5c,0xc5,0x4e,0xca,0x2b,0x4c,0x21,0x55,0xd3,0x94,0xac,0x38,0x5b,0x0d,0x2b,0x95,0x35,0xe2,0xa6,0xc9,0xbd,0xdd,0x56,0x89,0x0b,0xc7,0x9c,0x2d,0xd9,0x2e,0xd6,0xd2,0x77,0xda,0x64,0xbb,0x20,0x63,0xf3,0x68,0x4f,0x6d,0x80,0x25,0x5d,0x05,0x70,0x62,0x18,0x02,0x56,0x4a,0x03,0x00,0x80,0x07};
-const uint8_t spGOOD[] PROGMEM = {0x0A,0x28,0xCD,0x34,0x20,0xD9,0x1A,0x45,0x74,0xE4,0x66,0x24,0xAD,0xBA,0xB1,0x8C,0x9B,0x91,0xA5,0x64,0xE6,0x98,0x21,0x16,0x0B,0x96,0x9B,0x4C,0xE5,0xFF,0x01};
```

Comment:
Why is this removing GOOD and MORNING?

Reply:
Well, I had just added them in as a personal perk, but stopped using them. I suppose they could stay since they don't take up any memory if unused and are then there if somebody ever wanted them?

Reply:
Now I remember. There's no Male voice equivalent, so I thought it best to just remove.